scapy/layers/can.py

```diff
@@ -544,7 +558,10 @@ def read_packet(self, size=CAN_MTU):
 
         if is_log_file_format:
             t_b, intf, f = line.split()
-            idn, data = f.split(b'#')
+            if '##' in f:
+                idn, data = f.split(b'##')[0], f.split(b'##')[1][1:]
```

Comment:
why is this index access necessary? 
Line 576 and 577 clean up whitespaces

Reply:
Because CANFD frames from candump has an additional numerical value after the '##' which doesnt belong to the data e.g. 123##0AA. 
Im not sure what this is used for exactly, ignored it for now.

Reply:
Thanks. but still only one split in the code would be nicer and then a comment 

```
idn, data = f.split()
# removing something I dont know what it is
data = data[1:]
```